source/unit_data.h

```diff
@@ -0,0 +1,39 @@
+//file containing the unit data used for menu items.
+//this MUST be included after #include "..\global_defines.h"
+
+//arrays containing elements for menus
+//[menu_item, string, cost, unit_type, rank, skill_modifier]
+_menu_request_unit_items = [

```

Comment:
this is what I mentioned on the email, this way is simpler to add stuff to the menus, the costs can be set to numbers instead of #defines. all the changes here are applied automagically to the menu AND the spawn script